client/my-sites/email/email-different-domain-owner-message/index.tsx

```diff
@@ -0,0 +1,53 @@
+import { useTranslate } from 'i18n-calypso';
+import TrackComponentView from 'calypso/lib/analytics/track-component-view';
+import { recordTracksEvent } from 'calypso/lib/analytics/tracks';
+
+type EmailDifferentDomainOwnerMessageProps = {
+	source: string;
+};
+
+export const EmailDifferentDomainOwnerMessage = (
+	props: EmailDifferentDomainOwnerMessageProps
+) => {
+	const { source } = props;
+
+	const translate = useTranslate();
+
+	const onClickLink = ( eventType: 'login' | 'support' ) => {
```

Comment:
We could use a more explicit name for that event handler which better conveys its purpose, e.g. `recordClickEvent`.

Reply:
Also why do we need this parameter since `onClickLink()` is only called in one place?

Reply:
Done